test/OpenJ9_Jsr_292_API/src/com/ibm/j9/jsr292/ChildCustomClassLoader.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2001, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code is also Distributed under one or more Secondary Licenses,
+ * as those terms are defined by the Eclipse Public License, v. 2.0: GNU
+ * General Public License, version 2 with the GNU Classpath Exception [1]
+ * and GNU General Public License, version 2 with the OpenJDK Assembly
+ * Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *******************************************************************************/
+package com.ibm.j9.jsr292;
+
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Helper class loader used in JSR 292 tests for MethodHandles.Lookup API 
+ */
+public class ChildCustomClassLoader extends ParentCustomClassLoader {
```

Comment:
The implementation of this is identical to the ParentCustomClassLoader with only the name used in `loadClass` being different.

Is there a reason to duplicate the class and the implementations of each method?

Reply:
A single CustomClassLoader would be sufficient if passed 1) its parent & 2) the name of the "special class" to check in `loadClass`.

Reply:
All these files were directly copied from the original jsr292 directory without any changes.
So agreed and updated to use CustomClassLoader to avoid redundancy. 

I prefer not to change the same files in the original jsr292 directory (for java 8 and java 9 with b148) as these files are used by other tests which are irrelevant to our code.